src/app/components/login/login.component.ts

```diff
@@ -1,30 +1,44 @@
 import { Component, inject } from '@angular/core';
 import { FormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
-import { AuthService } from '../../services/auth.service';
 import { Router } from '@angular/router';
-import { SignupButtonComponent } from "../shared/signup-button/signup-button.component";
+import { SignupButtonComponent } from '../shared/signup-button/signup-button.component';
 import { NzFormModule } from 'ng-zorro-antd/form';
 import { NzInputModule } from 'ng-zorro-antd/input';
-
+import { Store } from '@ngrx/store';
+import { login } from '../../store/actions/auth.actions';
+import { AuthState } from '../../store/state';
 
 @Component({
-    selector: 'app-login',
-    standalone: true,
-    templateUrl: './login.component.html',
-    styleUrl: './login.component.css',
-    imports: [ReactiveFormsModule, SignupButtonComponent, NzFormModule, NzInputModule]
+  selector: 'app-login',
+  standalone: true,
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.css',
+  imports: [
+    ReactiveFormsModule,
+    SignupButtonComponent,
+    NzFormModule,
+    NzInputModule,
+  ],
 })
 export class LoginComponent {
   fb = inject(FormBuilder);
-  authService = inject(AuthService);
-  router =  inject(Router);
+  router = inject(Router);
 
   form = this.fb.nonNullable.group({
-    email: ['', Validators.required],
-    password: ['', Validators.required]
+    email: ['', Validators.required, Validators.email],
+    password: ['', Validators.required, Validators.minLength(6)],
```

Comment:
I was getting some errors where my Validators would not work unless I wrapped them in an array... so just fyi incase that happens to you too...